### PR TITLE
cadence 0.8.2

### PR DESCRIPTION
**Pull request by gromgit**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by dtrodrigues — COMMENTED** [review]
Can you squash your commits?

**Comment by gromgit** [comment]
Version bump + fix for test case failure in #60294

File: Formula/cadence.rb [hunk 1]
```diff
@@ -21,10 +21,10 @@ def install
...
   test do
     (testpath/"hello.cdc").write <<~EOS
-      pub fun main() {
-        log("Hello, world!")
+      pub fun main(): Int {
+        return 0
       }
     EOS
-    assert_match "Hello, world!", shell_output("#{bin}/cadence hello.cdc")
+    system "#{bin}/cadence", "hello.cdc"
```
Comment (chenrui333) [inline Formula/cadence.rb:28]: can we assert the output?

Comment (gromgit) [reply Formula/cadence.rb:28]: I took out the `log` call, so there's no output to assert. I've opened an issue with upstream to advise on why `log` is causing the panic: https://github.com/onflow/cadence/issues/361

**Review by dawidd6 — APPROVED** [review]

File: Formula/cadence.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/cadence.rb:28]: 👍

**Pull request closed without merging**